appendix_meta_rules.md

```diff
@@ -536,29 +543,36 @@ To keep the file names interoperable use the following:
 
 As a best practice use the prefix `mr_filter_`
 
+<!-- TODO: finalize the rx schema-->
 ### Schema
 
 ```yaml
 title: //str
 description: //str
 action: //str
 type: //str
-rules: //map
 logsource:
     product: //str
     service: //str
-selection:
+global_-_filter:
     type: //rec
+    required:
+      detection: //rec
+    optional:
+      <selections> // rec
+      rules: //arr # no defined rules means matching on all rules
+
 ```
 
 ### Syntax
 
-Like Sigma rules, "Filter" rules have a title.  
+Like Sigma rules, "Filter" rules have a title.
 They don't have an id or level as they use the one from the referenced rules.
+<!-- I think they should have a id, even if we don't use it (yet) -->
```

Comment:
Global Filter must not change the id of the original rules.
Can a have a `name` like correlation.

Reply:
The id of a global filter would not replace the id of the original rule. But it would be used to address the global filter, if needed.

Reply:
OK for me, But it has to be explained.

Reply:
+1 for ids. Having unique identifier for objects is always useful